README.md

```diff
@@ -2,6 +2,12 @@
 
 Web user interface for Batch Analysis
 
+## Contributing
+
+For compliance reasons, all pull requests **must** be submitted with a Jira ID in the pull request title.
+Include the Jira ID at the beginning of the title, enclosed in square brackets.
```

Comment:
TOL: maybe we can remove the requirement of "enclosed in square brackets". I don't think that's mandatory no?

Reply:
oops, you're right!